test/components/share_links_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'component_test_helper'
+
+class ShareLinksTest < ComponentTestCase
+  def component_name
+    "share-links"
+  end
+
+  test "fails to render share links when no share_urls are given" do
```

Comment:
Do we need to better handle the case where only one share link is provided? eg Facebook only?